core/domain/config_domain.py

```diff
@@ -220,31 +230,31 @@ def __init__(self, name, schema, description, default_value):
         Registry.init_config_property(self.name, self)
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Returns the name of the configuration property."""
 
         return self._name
 
     @property
-    def schema(self):
+    def schema(self) -> Dict[str, Any]:
```

Comment:
Explain why this returns `Any`

Reply:
schemas have key-value pair with strings as key and value can be any of these (https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L115
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L64 
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L33

Reply:
ditto as above

Reply:
done!

Reply:
done!